lib/layout/dummyLayoutV1.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * dummyLayoutV1.js
+ *
+ * this dumps a fake graph
+ *
+ * @author John O'Brien. 
+ */
+
+// Create a graph instance.
+
+
+function buildGraph() {
+  var lib = './../';
```

Comment:
Also, all of the these imports should probably be done at the top level of the file rather than inside of a graph constructor.